ppa/local_settings.py.sample

```diff
@@ -18,7 +18,7 @@ ALLOWED_HOSTS = []
 
 # SECURITY WARNING: keep the secret key used in production secret!
 # http://www.miniwebtool.com/django-secret-key-generator/
-SECRET_KEY = ''
+SECRET_KEY = '123456' # CHANGE ME !
```

Comment:
This tripped me up for a few minutes and was a concern in #273. This isn't a security concern, correct?

Reply:
It probably isn't much of a security concern, but it's not great, and some of the automated code tools we use will flag it and complain. Let's leave it blank, but add a note in the installation steps to explicitly state that you should set a secret key. 

Are there any other local settings we don't mention in the installation instructions that we should call out that need editing or review?